functests/0_config/config.go

```diff
@@ -173,4 +175,13 @@ func testProfile() *performancev2.PerformanceProfile {
 			},
 		},
 	}
+	// If the machineConfigPool is master, the automatic selector from PAO won't work
+	// since the machineconfiguration.openshift.io/role label is not applied to the
+	// master pool, hence we put an explicit selector here.
+	if utils.RoleWorkerCNF == "master" {
```

Comment:
works for me, I hope it just will not create us additional problems in the future

Reply:
The other option was to handle this special case in a generic way, doing way more parsing with higher risk of mistake.
The master case is really a special one, so this felt a lot safer.